### PR TITLE
2.4.14 released

### PR DESCRIPTION
**Pull request by johanandren**
DONT MERGE YET, needs an url to HTTP release first.

### CODE REVIEW

**Comment by ktoso** [comment]
Note for users who might find this before we announce – don't jump on it yet, thanks.

File: docs/index.md [hunk 1]
```diff
@@ -53,9 +53,7 @@ All artifacts are available on Maven Central.
 
 ### Akka HTTP
 
-Note that Akka HTTP is detached from the core Akka modules with it's own versioning scheme soon to come.
-This currently means you must depend on the HTTP 2.4.11 artifacts together with Akka 2.4.13 as there is
-no 2.4.13 HTTP artifacts.
+Akka HTTP maintains its own release cycle. The current stable series is <code>10.x</code>, and it is compatible with Akka 2.x.
```
Comment (johanandren) [inline docs/index.md]: wat, code-tag??

File: docs/index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ title: Akka Documentation
 
 ## Release Versions
 
-### Akka 2.4.12 (current stable release) for Scala 2.11 / 2.12.0-RC2 and Java 8+
+### Akka 2.4.14 (current stable release) for Scala 2.11 / 2.12.0 and Java 8+
```
Comment (ktoso) [inline docs/index.md]: right

**Comment by johanandren** [comment]
LGTM!

I wanted to approve but "pull request authors cannot review their own PRs".

File: docs/index.md [hunk 3]
```diff
@@ -53,9 +53,7 @@ All artifacts are available on Maven Central.
 
 ### Akka HTTP
 
-Note that Akka HTTP is detached from the core Akka modules with it's own versioning scheme soon to come.
-This currently means you must depend on the HTTP 2.4.11 artifacts together with Akka 2.4.12 as there is
-no 2.4.12 HTTP artifacts.
+Akka HTTP maintains its own release cycle. The current stable series is <code>10.x</code>, and it is compatible with Akka 2.x.
```
Comment (ktoso) [inline docs/index.md]: True, in fact we may depend on fixes or additions added in minor versions... how should I rephrase this?

Comment (patriknw) [inline docs/index.md]: "latest Akka 2.4.x" is short and correct

**Pull request merged**